src/breadcrumb-group/item/item.tsx

```diff
@@ -58,7 +61,7 @@ const BreadcrumbItemWithPopover = <T extends BreadcrumbGroupProps.Item>({
               arrow={position => (
                 <div className={clsx(popoverStyles.arrow, popoverStyles[`arrow-position-${position}`])}>
                   <div className={popoverStyles['arrow-outer']} />
-                  <div className={popoverStyles['arrow-inner']} />
+                  <div className={clsx(popoverStyles['arrow-inner'], isVisualRefresh && popoverStyles.refresh)} />
                 </div>
```

Comment:
Can we change this code and the one below to use the `<Arrow>` component from the previous PR: https://github.com/cloudscape-design/components/blob/b4da3b8e1a16b4eb746e1f603dce3b49175c633d/src/popover/arrow.tsx#L14-L23

Reply:
```suggestion
                <Arrow position={position} />
```

Reply:
That makes sense. I'll do that. Thanks for the suggestion.